### DISCUSSION TITLE
DatePicker & DateTimePicker 范围选择，希望支持快速选择年、月

### DISCUSSION THREAD

**Discussion by haodaking** [element-plus/element-plus#4200] [Ideas]
问题描述：
DatePicker & DateTimePicker 这两个组件 `type` 为范围选择的时候，希望支持快速选择年、月，同非范围选择时候的交互，可以点击 ”年份“和”月份”来快速选择。
目前只能通过左右箭头来切换年月，如果范围较广的时候就需要点击很多很多次。

业务场景：
如：选择一个人的入学时间和毕业时间。现在的做法是采用两个 datepicker 来解决的。

相关组件：
https://element.eleme.io/#/zh-CN/component/date-picker
https://element.eleme.io/#/zh-CN/component/datetime-picker

**Comment by lovesL** [comment]
不知道哪来那么多星星   真不好用  iss回复爱答不理

**Reply by 2350568089** [reply]
加了兄弟，type="dateRange"

**Comment by CrabR** [comment]
2025年了，这个好像还没加
